core/core_modules/issue.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * Creates an issue on github for kassy.
+ *
+ * Written By:
+ * 		Dion Woolley
+ *
+ * License:
+ *		MIT License. All code unless otherwise specified is
+ *		Copyright (c) Dion Woolley and Contributors 2016.
+ */
+
+var command = "issue",
+    githubAPI = "https://api.github.com/repos/mrkno/Kassy/issues",
+    token = 'ZWZkN2ZiMGY2OWViZDlkNmQyZmE0Yzc4ODFjNTkyY2JmNmQxYTljNA==',
+    request = require.safe('request'),
+    fs = require('fs'),
+    config = null;
+
+try {
+    config = require.once('../../config.json');
+}
+catch (e) {
+    //If config file does not exist.
```

Comment:
Feels like there should be a better way. Could look at how config.js loads files because AFAIK it doesn't use require.